packages/webapp/src/components/Form/Switch/index.tsx

```diff
@@ -22,8 +22,8 @@ import { Main } from '../../Typography';
 interface SwitchProps {
   checked?: boolean;
   onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
-  label?: string;
-  leftLabel?: string;
+  label?: ReactNode | string;
+  leftLabel?: ReactNode | string;
```

Comment:
If you end up adding the formatted labels here then this has the benefit of just being a string again.

Reply:
I actually prefer the more flexible type! It leaves open the possibility of passing one of those styled strings that requires a `<Trans />` component -- we haven't seen it yet with switches but the new form buttons had an example.